tensorflow_graphics/rendering/camera/perspective.py

```diff
@@ -201,40 +224,43 @@ def intrinsics_from_matrix(matrix, name=None):
       "perspective_intrinsics_from_matrix".
 
   Returns:
-    Tuple of two tensors, each one of shape `[A1, ..., An, 2]`. The first
-    tensor represents the focal length, and the second one the principle point.
+    Tuple of three tensors, first two of shape `[A1, ..., An, 2]` and the third of shape `[A1, ..., An, 1]`. The first
+    tensor represents the focal length, and the second one the principle point. The third represent Skew Coefficient.
 
   Raises:
     ValueError: If the shape of `matrix` is not supported.
   """
-  with tf.compat.v1.name_scope(name, "perspective_intrinsics_from_matrix",
-                               [matrix]):
-    matrix = tf.convert_to_tensor(value=matrix)
+  with tf.compat.v1.name_scope(name, "perspective_intrinsics_from_matrix", [matrix]):
+      matrix = tf.convert_to_tensor(value=matrix)
 
-    shape.check_static(
-        tensor=matrix,
-        tensor_name="matrix",
-        has_rank_greater_than=1,
-        has_dim_equals=((-1, 3), (-2, 3)))
+      shape.check_static(
+          tensor=matrix,
+          tensor_name="matrix",
+          has_rank_greater_than=1,
+          has_dim_equals=((-1, 3), (-2, 3)),
+      )
+
+      fx = matrix[..., 0, 0]
+      fy = matrix[..., 1, 1]
+      cx = matrix[..., 0, 2]
+      cy = matrix[..., 1, 2]
+      skew_coefficient = matrix[..., 0, 1]
```

Comment:
I'd just call this "skew" instead of "skew_coefficient", since I feel the "coefficient" is implied (here and elsewhere in this file).

Reply:
Done